pkgs/tools/system/osquery/default.nix

```diff
@@ -0,0 +1,64 @@
+{ libcxxStdenv, lib, fetchurl, fetchFromGitHub, pkgconfig, git,
+  cmake, python3, which, flex, bison, perl, libunwind, openssl,
+  buildAws ? false, buildBpf ? true
+}:
+
+libcxxStdenv.mkDerivation rec {
+  pname = "osquery";
+  version = "4.9.0";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
```

Comment:
Half of those packages are critical and receive somewhat often security fixes and should be used from nixpkgs.

Reply:
I spent some time attempting this, I feel like the reason this package was removed in the first place is it kept breaking when the dependencies changed..